custom-namespaces.yaml

```diff
@@ -268,6 +268,19 @@ contextual_services:
       type: ping_explore
       views:
         base_view: topsites_impression_firefox_ios
+contextual_services:
+  glean_app: false
+  connection: bigquery-oauth
```

Comment:
is oauth necessary for these tables?

Users will need to authenticate to BigQuery for any dashboard tile that uses them. It looks like the one table listed here is accessible to Mozilla. So we could remove `connection` here so that any access is done over the Looker service account (and users don't need to authenticate).

```suggestion
```